### PR TITLE
Fix slack notifier

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/content/pull/17321

## Description
Fix slack notifier empty messages bug

### CODE REVIEW

**Review by Noy-Maimon — APPROVED** [review]

File: Tests/scripts/gitlab_slack_notifier.py [hunk 1]
```diff
@@ -13,8 +13,9 @@
 DEMISTO_GREY_ICON = 'https://3xqz5p387rui1hjtdv1up7lw-wpengine.netdna-ssl.com/wp-content/' \
                     'uploads/2018/07/Demisto-Icon-Dark.png'
 ARTIFACTS_FOLDER = os.getenv('ARTIFACTS_FOLDER', './artifacts')
-ENV_RESULTS_PATH = os.getenv('ENV_RESULTS_PATH', os.path.join(ARTIFACTS_FOLDER, 'env_results.json'))
-PACK_RESULTS_PATH = os.path.join(ARTIFACTS_FOLDER, BucketUploadFlow.PACKS_RESULTS_FILE)
+ARTIFACTS_FOLDER_XSOAR = os.getenv('ARTIFACTS_FOLDER_XSOAR', './artifacts')
+ENV_RESULTS_PATH = os.getenv('ENV_RESULTS_PATH', os.path.join(ARTIFACTS_FOLDER_XSOAR, 'env_results.json'))
+PACK_RESULTS_PATH = os.path.join(ARTIFACTS_FOLDER_XSOAR, BucketUploadFlow.PACKS_RESULTS_FILE)
```
Comment (avidan-H) [inline Tests/scripts/gitlab_slack_notifier.py:18]: Why `ARTIFACTS_FOLDER_XSOAR`? Won't it be the case that sometimes you'll be reporting on the mpv2 packs?

File: Tests/scripts/gitlab_slack_notifier.py [hunk 2]
```diff
@@ -47,7 +48,7 @@ def options_handler():
     return options
 
 
-def get_artifact_data(artifact_relative_path: str) -> Optional[str]:
+def get_artifact_data(artifact_folder, artifact_relative_path: str) -> Optional[str]:
```
Comment (avidan-H) [inline Tests/scripts/gitlab_slack_notifier.py:51]: Need to update the docstring.

File: Tests/scripts/gitlab_slack_notifier.py [hunk 3]
```diff
@@ -73,7 +74,7 @@ def get_artifact_data(artifact_relative_path: str) -> Optional[str]:
 def get_fields():
     failed_tests = []
     # failed_tests.txt is copied into the artifacts directory
-    failed_tests_file_path = os.path.join(ARTIFACTS_FOLDER, 'failed_tests.txt')
+    failed_tests_file_path = os.path.join(ARTIFACTS_FOLDER_XSOAR, 'failed_tests.txt')
```
Comment (avidan-H) [inline Tests/scripts/gitlab_slack_notifier.py:77]: There will never be a case when this artifact will be in the other directory?

**Review by avidan-H — CHANGES_REQUESTED** [review]

File: Tests/scripts/gitlab_slack_notifier.py (hunk 1 above)
Comment (yaakovi) [reply Tests/scripts/gitlab_slack_notifier.py:18]: we don't have that flow ready yet, so no

File: Tests/scripts/gitlab_slack_notifier.py (hunk 3 above)
Comment (yaakovi) [reply Tests/scripts/gitlab_slack_notifier.py:77]: this is the nightly flow which only runs against XSOAR

**Review by avidan-H — APPROVED** [review]

**Pull request merged**